draft-ietf-taps-arch.md

```diff
@@ -214,7 +214,7 @@ Sockets, for protocols like TCP, are generally limited to connecting to a single
 
 Transport Services implementations are meant to be flexible at connection establishment time, considering many different options and trying to select the most optimal combinations ({{gathering}} and {{racing}}). This requires applications to provide higher-level endpoints than IP addresses, such as hostnames and URLs, which are used by a Transport Services implementation for resolution, path selection, and racing.
 
-Flexibility after connection establishment is also important. Transport protocols that can migrate between multiple network-layer interfaces need to be able to process and react to interface changes. Protocols that support multiple application-layer streams need to support initiating and receiving new streams using existing connections.
+Flexibility after connection establishment is also important. Transport protocols that can migrate between multiple network-layer interfaces need to be able to process and react to interface changes. Protocols that support multiple application-layer streams need to support initiating and receiving new streams using existing connections. Transport services implementations can further implement fallback mechanisms if connection establishment of one protocol fails or performance is detected to be unsatisfactory.  
```

Comment:
I like the sentence, but I'm not sure that it fits in this paragraph. This new sentence mentions a protocol failing establishment, but the paragraph begins `Flexibility after connection establishment...`.

Rather, this feels like it belongs with the previous paragraph, `Transport Services implementations are meant to be flexible at connection establishment time...`

Reply:
Done.